services/multitenant/tenantstats_test.go

```diff
@@ -35,6 +36,10 @@ var (
 	workspaceID3 = uuid.Must(uuid.NewV4()).String()
 )
 
+func almostEqual(a, b float64) bool {
+	return math.Abs(a-b) <= float64EqualityThreshold
+}
```

Comment:
I think that there shoud be a `require.InDelta()` function that does this.

Reply:
https://pkg.go.dev/github.com/stretchr/testify/require#InDelta

Reply:
Let me check this out and incorporate this